layouts/community/ergodox/drashna/keymap.c

```diff
@@ -401,11 +401,10 @@ void suspend_wakeup_init_keymap(void) {
     rgb_matrix_set_suspend_state(false);
 }
 
-void rgb_matrix_layer_helper (uint8_t red, uint8_t green, uint8_t blue) {
-  rgb_led led;
+extern led_config_t g_led_config;
```

Comment:
If this is going to be used a lot, it may be worth adding it to the rgb matrix header, so that it can be easily referenced.

Reply:
ran into a compile issue with the new struct in the header as an extern define. Not sure why, will follow up with another pr making this stuff easier to use in keymaps.

Reply:
okay. And very odd.